Formula/petsc-complex.rb

```diff
@@ -0,0 +1,41 @@
+class PetscComplex < Formula
+  desc "Portable, Extensible Toolkit for Scientific Computation (with complex support)"
```

Comment:
TestBot is failing due to brew audit; the formula description is 93 chars but needs to be ,= 80. Maybe instead of `(with complex support)` we could say `(complex)`? Anyone else have thoughts on how to appease brew audit?